src/components/header/headerSearchBar/SearchBar.tsx

```diff
@@ -23,53 +38,69 @@ export interface SearchRequest {
 }
 
 export const SearchBar: React.FC<SearchBarProps> = ({ example, resetExample, closeMenu }) => {
-  const [value, setValue] = useState("");
-  const [active, setActive] = useState(false);
-  const [searchResults, setSearchResults] = useState<SearchRequest[]>([]);
-  const { storedValue: searchDupResults, setValue: setSearchDupResults } = useLocalStorage<
-    SearchRequest[]
-  >(SEARCH_HISTORY, []);
+  const [searchBar, setSearchBar] = useRecoilState(searchBarAtom);
```

Comment:
are addresses stored to local storage by any other mechanism after this change? (i.e. we shouldn't remove that functionality)

Reply:
no, mechanism to put data in local storage remains the same

Reply:
where is the code to use local storage? because here I only see a removal...

also -- if this comment is correct (https://github.com/orbs-network/ton-contract-verifier/pull/82#discussion_r1041875886), we should do the same here